app/controllers/api/v1/likes_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    # Controller class that is responsible for handling like requests.
+    class LikesController < ApplicationController
+      include DeviseTokenAuth::Concerns::SetUserByToken
+      include Pundit
+
+      before_action :authenticate_user!
+      before_action :like, only: %i[destroy]
+      before_action :authorize_like!
+      after_action :verify_authorized
```

Comment:
should be before

Reply:
I was wrong
https://github.com/varvet/pundit#ensuring-policies-and-scopes-are-used